client/src/components/Collections/PairedListCollectionCreator.test.js

```diff
@@ -29,6 +30,82 @@ describe("PairedListCollectionCreator", () => {
 
     it("autopairs the dataset", async () => {
         // Autopair is called on startup
-        expect(wrapper.findAll("li.dataset").length == 0).toBeTruthy();
+        expect(wrapper.findAll("li.dataset unpaired").length == 0).toBeTruthy();
+    });
+
+    it("selects the correct name for an auotpair", async () => {
+        wrapper = mount(PairedListCollectionCreator, {
+            propsData: {
+                initialElements: DATA._2,
+                creationFn: () => {
+                    return;
+                },
+                oncreate: () => {
+                    return;
+                },
+                oncancel: () => {
+                    return;
+                },
+                hideSourceItems: false,
+            },
+        });
+        await wrapper.vm.$nextTick();
+        //change filter to .1.fastq/.2.fastq
```

Comment:
Seems like the default filter should include the . then? Can we add a test case for the default and keep this test case for the changing the filter?

Reply:
The "autopairs the datasets" test case uses the default filter _1/_2.